modules/kubernetes-addons/variables.tf

```
@@ -9,6 +9,12 @@ variable "eks_cluster_domain" {
   type        = string
 }
 
+variable "eks_cluster_domains" {
```

I don't know that `eks_cluster_domains` shows the intent and use of the variable - since this is passed in to the cert-manager addon `domain_names` variable, I think its better if we just call it `cert_manager_domain_names`

```suggestion
variable "cert_manager_domain_names" {
```

---

I agree. It should be just a separate setting exclusively for cert-manager